### PR TITLE
Create notebook for `Auto` and `Tune`

### LINKED ISSUE #1274: Create notebook for `Auto` and `Tune`
### 🚀 Feature Request

We should show how `Auto` and `Tune` work in our tutorials.

Current script with example isn't shown in our documentation.

### Proposal

Create a notebook with examples of usage of `Auto` and `Tune`. What should be covered:
- What `Tune` class does
  - How it works
    - What kind of hyperparameters are tried, `params_to_tune` method
    - How to write custom `params_to_tune` and check them
    - Runner: 
    - Storage: how to suspend and continue the tuning using storage
  - Example of tuning with some pipeline
  - Analysis
    - `summary`
    - `top_k`
- What `Auto` class does
  - How it works
    - Pool stage
    - Tuning stage
    - How stages are working inside `fit`, how resources are divided between them
  - Example of tuning with default pool

### Test cases

-

### Additional context

Blocked by #1269, #1273.

### PR DESCRIPTION
**Pull request by Mr-Geekman**
## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [x] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Proposed Changes
1. Skip duplicates during optimization.
2. Remove duplicates from top_k
3. Look #1274.

## Closing issues
Closes #1274

### CODE REVIEW

File: examples/etna-auto.db
Comment (martins0n) [inline examples/etna-auto.db:1]: Do we need this?

**Review by martins0n — CHANGES_REQUESTED** [review]

**Comment by martins0n** [comment]
https://github.com/tinkoff-ai/etna/pull/1285#issuecomment-1588709418

Did you find reason?  In what place does it happen?

File: examples/etna-auto.db
Comment (Mr-Geekman) [reply examples/etna-auto.db:1]: I will remove it.

**Comment by Mr-Geekman** [comment]
<p>I tried to do this in the next block. What do you mean by showing only important columns? Change existing <code>summary</code> method or limit columns in this block and don't show full <code>sammary</code> ?  </p>

---

 <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1285/discussion/#comment-1589291896'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by Mr-Geekman** [comment]
<p>It is easy to see at the summary by the column <code>study</code>. Top rows are with <code>tune/</code> values, not with <code>pool</code>.I can emphasize it in the text.</p><p> </p><p>Holdout is a good idea, I will try to add it.     </p>

---

 <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/1285/discussion/#comment-1589291930'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Pull request merged**